### PR TITLE
Lesson 01

### PR DESCRIPTION
**Pull request by AlexPFrank**
The first lesson is done :)

### CODE REVIEW

File: Task1.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		<title>PAGE FOR TASK 1</title>
+		<meta charset="UTF-8">
+		<style>
+			.underline{
+				text-decoration: underline;
+			}
+			.linethrough{
+				text-decoration: line-through;
+			}
+			.ac{
```
Comment (timchukd) [inline Task1.html]: Please do not use short variables name in the code. We are writing the code for the each other and good when from the first look is clear what the developer mean. Optimization stuff like make from long variable names -> short is job of builders.

File: Task1.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+
+	<head>
+		<title>PAGE FOR TASK 1</title>
+		<meta charset="UTF-8">
+		<style>
+			.underline{
```
Comment (timchukd) [inline Task1.html:7]: this class is not use in your code

File: Task1.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+<html>
+
+	<head>
+		<title>PAGE FOR TASK 1</title>
+		<meta charset="UTF-8">
+		<style>
+			.underline{
+				text-decoration: underline;
+			}
+			.linethrough{
```
Comment (timchukd) [inline Task1.html:10]: this class is not use in your code

File: Task1.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+				text-decoration: line-through;
+			}
+			.ac{
+				text-align: center;
+			}
+			.ar{
+				text-align: right;
+			}
+			.ad{
```
Comment (timchukd) [inline Task1.html]: this class is not use in your code

File: Task1.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (timchukd) [inline Task1.html]: unneeded line

File: Task1.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+			.ad{
+				text-align: justify;
+			}
+		</style>
+	</head>
+	
+	<body>
+		<h1 class="ac">TASK 1</h1>
+		<hr/>
+		<p>It's a text, that I've prepared for Task-1. This sentence is used for example of <strong>bold</strong> style of words. This sentence is used for example of <em>curve</em> style of words. This sentence is used for example of <a class="underline">underlined</a> style of words. And this sentence is used for example of <a class="linethrough">NOT</a> linedthrough style of words.<p>
```
Comment (timchukd) [inline Task1.html]: please do not write everything in one line
limit is 100 characters (this is +/- the width of the screen)

I guess this way looks better for the reader:
```
<p>
    It's a text, that I've prepared for Task-1. This sentence is used for example of 
    <strong>bold</strong> style of words. This sentence is used for example of 
    <em>curve</em> style of words. 
    This sentence is used for example of <a class="underline">underlined</a> style of words. 
    And this sentence is used for example of <a class="linethrough">NOT</a> linedthrough style of words.
<p>
```

P.S. please check code in the bottom

File: Task2.html [hunk 7]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (timchukd) [inline Task2.html]: unneeded space

File: Task2.html [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+	</body>
+		<img src="http://www.kurzweilai.net/images/Naam-Limits-of-Earth-Part1-001-earth-600x600.jpg" height="400" width="400" style="float: left" style="padding: 7px 7px 7px 7px"><p>It's a text, which I've also prepared for Task-1, but here will be the third part of the task. Here will be shown how to do text align to the center. It's a text, which I've also prepared for Task-1, but here will be the third part of the task. Here will be shown how to do text align to the center. It's a text, which I've also prepared for Task-1, but here will be the third part of the task. Here will be shown how to do text align to the center.</p>
```
Comment (timchukd) [inline Task2.html]: this tag should be inside body

File: Task3.html [hunk 9]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (timchukd) [inline Task3.html]: unneeded line

File: Task3.html [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+			.ac{
+				text-align: center;
+			}
+		</style>
+	</head>
+	
+	<body>
+		<h1 class="ac">Task 3</h1>
+		<hr/>
+		<h3><a style="text-decoration: underline">It's a first part of the task 3:</a> SO<sub>2</sub> + NaOH = Na<sub>2</sub>SO<sub>3</sub> + H<sub>2</sub>O</h3>
```
Comment (timchukd) [inline Task3.html:16]: please do not write all in one line
p.s. please check rest of the tasks and fix it

File: Task4.html [hunk 11]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (timchukd) [inline Task4.html]: unneeded line

File: Task4.html [hunk 12]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+
+	<head>
+		<title>PAGE FOR TASK 4</title>
+		<meta charset="UTF-8">
+		<style>
+			.ac{
```
Comment (timchukd) [inline Task4.html]: please check previous comment about short variable name

**Review by timchukd — CHANGES_REQUESTED** [review]
good job! but please check my comments :)

**Comment by AlexPFrank** [comment]
OK, I'll fix it as soon as I'll be able to do it

File: Task1.html (hunk 2 above)
Comment (AlexPFrank) [reply Task1.html:7]: This class is used in the first paragraph (<p>........<a = class"underline">.........</a>..........</p>)

File: Task1.html (hunk 3 above)
Comment (AlexPFrank) [reply Task1.html:10]: This class is also used in the same paragraph like class "underline"

File: Task1.html (hunk 4 above)
Comment (AlexPFrank) [reply Task1.html]: It was fixed

File: Task1.html (hunk 5 above)
Comment (AlexPFrank) [reply Task1.html]: It was fixed

File: Task2.html (hunk 8 above)
Comment (AlexPFrank) [reply Task2.html]: It was fixed

File: Task3.html (hunk 9 above)
Comment (AlexPFrank) [reply Task3.html]: It was fixed

File: Task3.html (hunk 10 above)
Comment (AlexPFrank) [reply Task3.html:16]: It was fixed

File: Task4.html (hunk 11 above)
Comment (AlexPFrank) [reply Task4.html]: It was fixed

File: Task4.html (hunk 12 above)
Comment (AlexPFrank) [reply Task4.html]: It was fixed

File: Task1.html (hunk 1 above)
Comment (AlexPFrank) [reply Task1.html]: It was fixed

File: Task1.html (hunk 2 above)
Comment (timchukd) [reply Task1.html:7]: ok

File: Task1.html (hunk 3 above)
Comment (timchukd) [reply Task1.html:10]: ok

**Review by timchukd — APPROVED** [review]

**Comment by timchukd** [comment]
good to go! 👍 

p.s. "It was fixed": you could write just "fixed" it means the same

**Pull request merged**